examples/gaiacli/client.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import "github.com/spf13/cobra"
+
+const (
+	// these are needed for every init
+	flagChainID = "chain-id"
+	flagNode    = "node"
+
+	// one of the following should be provided to verify the connection
+	flagGenesis = "genesis"
+	flagCommit  = "commit"
+	flagValHash = "validator-set"
+
+	flagSelect = "select"
+	flagTags   = "tag"
+	flagAny    = "any"
+
+	flagBind      = "bind"
+	flagCORS      = "cors"
+	flagTrustNode = "trust-node"
+
+	// this is for signing
+	flagName = "name"
+)
+
+var (
+	statusCmd = &cobra.Command{
+		Use:   "status",
+		Short: "Query remote node for status",
+		RunE:  todoNotImplemented,
+	}
+)
+
+// AddClientCommands returns a sub-tree of all basic client commands
+//
+// Call AddGetCommand and AddPostCommand to add custom txs and queries
+func AddClientCommands(cmd *cobra.Command) {
+	cmd.AddCommand(
+		initClientCommand(),
+		statusCmd,
+		blockCommand(),
+		validatorCommand(),
+		lineBreak,
+		txSearchCommand(),
+		txCommand(),
+		lineBreak,
+	)
+}
+
+// GetCommands adds common flags to query commands
+func GetCommands(cmds ...*cobra.Command) []*cobra.Command {
+	for _, c := range cmds {
+		c.Flags().Bool(flagTrustNode, false, "Don't verify proofs for responses")
```

Comment:
All flags should be begin with lowercase

Reply:
Huh? I thought your last comment said they should be upper case...

https://github.com/cosmos/cosmos-sdk/pull/349#pullrequestreview-90294919

I don't care. Just want consistency. Can you add this to the coding doc and I will follow.

Upper case. lower case. CamelCaseFlagDescription 

I don't care.

Reply:
https://github.com/tendermint/coding/blob/master/go/coding_standard.md#cli

Reply:
yeah lower case in coding standards